src/Groups/libraries/smallgroups.jl

```diff
@@ -83,7 +83,7 @@ Pc group of order 60
 ```
 """
 function small_group(::Type{T}, n::IntegerUnion, m::IntegerUnion) where T
-  G = _small_group(n, m)
+  G = small_group(n, m)
```

Comment:
Is this change intended? I am not deep enough into the details to understand the difference

Reply:
Yes. (It is also mentioned in the corresponding commit message.)